### ISSUE TITLE
Couldn't scan and decode MaxiCode Code

### ISSUE THREAD

**Issue by bestpower** [zxing-cpp/zxing-cpp#354]
In zxing-cpp\core\src\BarcodeFormat.h we can see it include MaxiCode, but the android app coundn't scan and output result.

**Comment by duanqn** [comment]
I think MaxiCode is not officially supported yet.

**Comment by axxel** [comment]
It is supported but the detector code is practically dysfunctional for anything other than `isPure` (perfectly aligned) situations. [Those examples](https://github.com/nu-book/zxing-cpp/tree/master/test/samples/maxicode-1) work, while [those](https://github.com/nu-book/zxing-cpp/tree/master/test/samples/maxicode-2) don't.

**Comment by bestpower** [comment]
Thanks. After test, It could scan and decode pure MaxiCode Code photo, but this situation only in developing stage, In actual using is sure to contain other uncorrelated areas,  if it couldn't overcome these interferences, this scanning maxicode's pragmatic value is relatively low.

**Comment by axxel** [comment]
> this scanning maxicode's pragmatic value is relatively low.

You are absolutely right. The Aztec detector is not quite as bad but also pretty limited in real-world situations. I will likely not have the time to fix that (unless someone offers me money ;)). I'll be happy to look at PR that improve the situation, though.

**Comment by axxel** [comment]
Same here, why do you consider this issue to be completed?
